app/src/app/pages/admin/shared/table/table.component.html

```diff
@@ -0,0 +1,40 @@
+<section class="admin_products">
+  <app-loader *ngIf="products.length === 0"></app-loader>
+  <div class="container">
```

Comment:
You have no logic when our products don't come in, btw the table is always visible. You can use something like this ' *ngIf="products.length !== 0" '